server/controller/gifController.js

```diff
@@ -11,54 +13,47 @@ cloudinary.config({
 })
 
 
-
-
-
-const createGif =  (req, res) =>{
-    try {
-      const data = {
-        title: req.body.title,
-        image: req.body.image
-      }
-        cloudinary.uploader.upload(data.image)
-    .then((image) => {
-      db.pool.connect((err, client, done) =>  {
-        const createdOn = new Date
-        //  i can also use *Date().toLocaleString();
-        
-        const values = [data.title, image.url, createdOn];
-        client.query(queries.createNewGif, values)
-          .then((result) => {
-            result = result.rows[0];
-
-            // send success response
-            res.status(201).send({
+class GifController {
+    static async createGif(req, res) {
+        try {
+            const { title, image} = req.body
+          let imageURL;
+          let publicId;
+        await cloudinary.uploader.upload(image, (err, result) => {
+            if (err) {
+                return res.status(500).send({
+                    status: "error",
+                    message: `Error uploading image`
+                })
+            }
+            imageURL = result.secure_url;
+            publicId = result.public_id
+        })
+         const created_at = new Date
+         const created_by = req.user.email;
```

Comment:
you don't need created_by if you have userId, what is the point? since we have the user id we can use that to identify who created the gif

Reply:
Ok sir, I would remove it